guides/v2.1/cloud/project/project-routes-more-redir.md

```diff
@@ -34,37 +36,35 @@ http://{default}/:
             "/regexp/(.*)/matching": { "to": "http://example.com/$1", regexp: true }
 ```
 
-This format is more rich and works with any type of route, including routes served directly by the application.
+You can use partial redirects with any type of route, including routes served directly by the application.
 
 Two keys are available under `redirects`:
 
--  `expires`: Optional, the duration that the redirect is cached.
+- `expires`—Optional, the duration that the redirect is cached.
     Examples of valid values include `3600s`, `1d`, `2w`, `3m`.
 
--  `paths`: The paths to apply the redirects to.
+- `paths`—The paths to apply the redirects to.
 
 The following defines each rule under `paths`:
 
--  A key that describes the expression to match against the request path
--  A value object describing both the destination to redirect to with detail on how to handle the redirection.
+- A key that describes the expression to match against the request path
+- A value object describing both the destination to redirect to with detail on how to handle the redirection.
 
 The value object is defined with the following keys:
 
--  [`to`](#cloud-route-partial-to)
--  [`regexp`](#cloud-route-partial-regexp)
--  [`prefix`](#cloud-route-partial-prefix)
--  [`append_suffix`](#cloud-route-partial-append)
--  [`code`](#cloud-route-partial-code)
+- [`to`](#cloud-route-partial-to)
```

Comment:
This is like a mini-TOC and is unnecessary since we have the right-hand navigation. These can be removed.